packages/nc-gui-v2/composables/useGridViewColumnWidth.ts

```diff
@@ -0,0 +1,67 @@
+import { useStyleTag } from '@vueuse/core'
+import type { ColumnType, GridColumnType } from 'nocodb-sdk'
+import type { Ref } from 'vue'
+import type GridView from '../../nocodb/src/lib/models/GridView'
```

Comment:
please don't import from '../../nocodb/...' it breaks our typecheck completely.
Just fall back to `any` typing or copy the interface for now because this isn't working right.